packages/gatsby-reporter/src/index.ts

```diff
@@ -12,3 +13,7 @@ patchConsole(reporter)
 export { reporter }
 export { setStore }
 export const reduxLogReducer = reducer
+
+// Types
+export { IActivityArgs, IPhantomReporter, IProgressReporter }
+export type IReporter = typeof Reporter
```

Comment:
Is this correct? Should it be `typeof reporter`. Or just `export type Reporter`

Reply:
No, Reporter is a class. So we have to get it's type via `typeof Reporter`, so this is correct